apps/src/aiTutor/types.ts

```diff
@@ -4,18 +4,17 @@ import {
   PiiTypes as PII,
 } from '@cdo/apps/util/sharedConstants';
 
-export type AITutorTypesValue =
-  (typeof AITutorTypes)[keyof typeof AITutorTypes];
-export type AITutorInteractionStatusValue =
-  (typeof AITutorInteractionStatus)[keyof typeof AITutorInteractionStatus];
+// TODO: Update this once https://codedotorg.atlassian.net/browse/CT-471 is resolved
+export type AITutorTypesValue = string;
+export type AITutorInteractionStatusValue = string;
```

Comment:
Functionally, this was only checking for a string before. Narrowing the type-checking here was difficult and throwing many errors because these are generated constants. I filed https://codedotorg.atlassian.net/browse/CT-471 to address.